.github/workflows/sync_fork.yml

```diff
@@ -0,0 +1,22 @@
+name: Sync fork
+
+on:
+  schedule:
+    - cron:  '0 0 * * *' 
+
+  workflow_dispatch: 
+
+jobs:
+  sync_with_upstream:
+    runs-on: ubuntu-latest
+    name: Sync master with upstream latest
+
+    steps:
+    - name: Checkout tetrate
```

Comment:
We would have to move the base branch to tetrate-workflow

Reply:
This shouldn't matter, cause this action is just for setting up the environment, though I need to remove that ref from `tetrate` since that branch is going to be removed

Reply:
Yup, to be consistent name it tetrate-workflow